src/sass/_advantages.scss

```diff
@@ -0,0 +1,146 @@
+@use './variables' as var;
+@use './global';
+
+.advantages {
+  background-color: #fff5f6;
+
+  &__container {
+    @extend .container;
+    padding-top: 39px;
+    padding-left: 20px;
+
```

Comment:
Dodaj tu proszę:
```
@media screen and (max-width: 767px) {
      max-width: 480px;
    }
```
Zapobiegnie to rozjeżdżaniu się układu pomiędzy 480 a 768px.